### PR TITLE
feat: support string concat in dynamic import analysis

### PR DESCRIPTION
**Pull request by lucacasonato**
`import("./foo/" + value)` is now supported in addition to ``import(`./foo/${value}`)``.

### CODE REVIEW

File: src/dep.rs [hunk 1]
```diff
@@ -292,14 +293,16 @@ impl<'a> Visit for DependencyCollector<'a> {
...
         } else {
           let mut parts =
             Vec::with_capacity(tpl.quasis.len() + tpl.exprs.len());
           for i in 0..tpl.quasis.len() {
-            if tpl.quasis[i].raw.len() > 0 {
-              parts
-                .push(DynamicTemplatePart::String(tpl.quasis[i].raw.clone()));
+            let cooked = tpl.quasis[i].cooked.as_ref().unwrap();
+            if cooked.len() > 0 {
+              parts.push(DynamicTemplatePart::String(cooked.clone()));
```
Comment (lucacasonato) [inline src/dep.rs:305]: @dsherret PTAL - the previous code was incorrect. Raw includes escape sequences in raw form.

Comment (lucacasonato) [inline src/dep.rs:305]: @dsherret PTAL - the previous code was incorrect. Raw includes escape sequences in raw form.

**Review by marvinhagemeister — APPROVED** [review]

File: src/dep.rs [hunk 2]
```diff
@@ -308,6 +311,47 @@ impl<'a> Visit for DependencyCollector<'a> {
           DynamicArgument::Template(parts)
         }
       }
+      Expr::Bin(bin) => {
+        let mut parts = Vec::with_capacity(2);
```
Comment (dsherret) [inline src/dep.rs:315]: Capacity is not necessarily correct here I think?

Comment (dsherret) [reply src/dep.rs:315]: Oh, I guess this is just allocating for the common case.

**Pull request merged**